feeds/npm/.golangci.yml

```diff
@@ -61,7 +61,7 @@ linters:
     - unused
     - varcheck
     - whitespace
-    - wrapcheck
+    # - wrapcheck
```

Comment:
Any reason for these to commented out?

Reply:
Because they were catching errors that existed prior to this refactor. My goal with this PR is just to reorganize things. I'll fix linting issues with the existing code in a future PR 😄